src/components/Header.module.scss

```diff
@@ -0,0 +1,7 @@
+.headerStyle {
+  padding: 10px;
+  height: 75px;
+  width: 100%;
+  background-image: linear-gradient(to right, #33b3a6, #40e0d0);
+  font-size: xx-large;
```

Comment:
Use numbers. So it is beyond clear what font-size it is. Also, if you care about accessibility, you should use rems -> https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/ but no need to worry about it from now. Let's use pixels only. Again, just mentioning so you know the different approaches

Reply:
understood, fixed